### PR TITLE
Fixed some compile warnings

### PR DESCRIPTION
**Pull request by ClarkPoon**

### CODE REVIEW

File: Apps/Src/Task_LogInfo.c [hunk 1]
```diff
@@ -12,7 +13,7 @@ void Task_LogInfo(void *p_arg) {
     EEPROM_LogData(DATA_SOC, Charge_GetPercent());
 
     //1 tick every 10 millisconds, so 300 ticks for a delay of 3000ms --> 3s
-    //OS_OPT_TIME_DLY specifies relative delay, so the delay is started from the �current time + dly�.
+    //OS_OPT_TIME_DLY specifies relative delay, so the delay is started from the �current time + dly�.
```
Comment (SijWoo) [inline Apps/Src/Task_LogInfo.c:16]: Is it just me or have these quotation marks been changed to diamonds with questions marks in them?

File: Apps/Src/Task_PetWDog.c [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #include "Tasks.h"
 #include "os.h"
 #include <stdlib.h>
```
Comment (SijWoo) [inline Apps/Src/Task_PetWDog.c:3]: Why was stdlib included? This slipped through from another PR :P

File: Apps/Src/Task_LogInfo.c (hunk 1 above)
Comment (ClarkPoon) [reply Apps/Src/Task_LogInfo.c:16]: I'll look into this

**Review by SijWoo — APPROVED** [review]
noice

**Pull request merged**